auth_backends/__init__.py

```diff
@@ -3,4 +3,4 @@
  These package is designed to be used primarily with Open edX Django projects, but should be compatible with non-edX
  projects as well.
 """
-__version__ = '0.5.3'  # pragma: no cover
+__version__ = '0.6.0'  # pragma: no cover
```

Comment:
That's a good point. I may make some OIDC/OAuth changes as part of the upcoming API scopes work. That's when we'll go to 1.0.0.